Network/Public/Include/Network/Connection.hpp

```
@@ -325,36 +317,33 @@ class Connection : public std::enable_shared_from_this<Connection>
         }
     }
 
-    suppressWarning(4100, "-Wunused-parameter")
```

Why did you remove the macro to ignore the error?